features/ct_alerts.feature

```diff
@@ -11,5 +11,3 @@ Feature: Alert Content type
   Scenario: Verify Alert JSON API
     When I go to "/jsonapi/node/alert"
     Then the response status code should be 200
-    And the "node_list" cache tag should not be used
```

Comment:
`node_list` should not be appearing so lets not uncomment that. If its appearing we have a bug I think.

Reply:
So this is on the JSON API,  because we remove the `AlertsJSONAPISubscriber.php`  then the node_list tag is not removed , do you want to instead restore this?

Reply:
Oh, its /jsonapi. I dont care about that anymore :). This is not an issue.